src/NuGet.Clients/NuGet.PackageManagement.UI/PackageLoadContext.cs

```diff
@@ -78,11 +77,10 @@ public IEnumerable<string> GetSupportedFrameworks()
                 else
                 {
                     // we also need to process SupportedFrameworks
-                    IEnumerable<NuGetFramework> supportedFrameworks;
-                    if (project.TryGetMetadata(
-                        NuGetProjectMetadataKeys.SupportedFrameworks,
-                        out supportedFrameworks))
+                    (bool supportedFrameworksSuccess, object supportedFrameworksValue) = await project.TryGetMetadataAsync<IEnumerable<NuGetFramework>>(NuGetProjectMetadataKeys.SupportedFrameworks, CancellationToken.None);
```

Comment:
Why is `supportedFrameworksValue` of type `object`?  `TryGetMetadataAsync<T>(...)` returns `(bool, T)`.  By using the expected type here you'd avoid an unnecessary cast [here](https://github.com/NuGet/NuGet.Client/pull/3485/files#diff-012bfa6ec396c4223ef75bc6de911ae7R83).